### ISSUE TITLE
C-= does not register

### ISSUE THREAD

**Issue by rogpld** [masm11/emacs#111]
(pgtk-use-im-context nil)

C-= works, single quote and accents do not work

(pgtk-use-im-context t)

C-= does not works, single quote and accents work

How can I get both working?

**Comment by masm11** [comment]
Does your single quote need special keys?

**Comment by rogpld** [comment]
No, I was using a traditional US-international with dead keys layout. Did not managed to get it working. If I'm not mistaken accents did not work as expected as well.
